handlers.ts

```diff
@@ -0,0 +1,61 @@
+import fs = require("fs");
+
+const filePath = "file.txt";
+
+
+async function getUpdateTime() {
+    return new Promise((resolve, reject) => fs.stat(filePath, (err, stats) => {
+            if (err) {
+                return reject(err);
+            }
+            resolve(stats.mtime.toLocaleString());
+        }
+    ));
+}
+
+async function sendUpdateTime(req, res) {
+    try {
+        res.writeHead(200, {
+            'Content-Type': 'text/plain'
+        });
+        let time = await getUpdateTime();
```

Comment:
by default variable should be `const` unless you change it

Reply:
done